viz_scripts/scaffolding.py

```diff
@@ -122,59 +152,94 @@ def load_viz_notebook_data(year, month, program, study_type, dynamic_labels, dic
     labeled_ct = filter_labeled_trips(participant_ct_df)
     expanded_ct = expand_userinputs(labeled_ct)
     expanded_ct = data_quality_check(expanded_ct)
+    expanded_ct = map_trip_data(expanded_ct, study_type, dynamic_labels, dic_re, dic_pur)
+
+    # Document data quality
+    file_suffix = get_file_suffix(year, month, program)
+    quality_text = get_quality_text(participant_ct_df, expanded_ct, None, include_test_users)
+
+    debug_df = pd.DataFrame.from_dict({
+            "year": year,
+            "month": month,
+            "Registered_participants": len(get_participant_uuids(program, include_test_users)),
+            "Participants_with_at_least_one_trip": unique_users(participant_ct_df),
+            "Participant_with_at_least_one_labeled_trip": unique_users(labeled_ct),
+            "Trips_with_at_least_one_label": len(labeled_ct),
+            "Trips_with_mode_confirm_label": trip_label_count("Mode_confirm", expanded_ct),
+            "Trips_with_trip_purpose_label": trip_label_count("Trip_purpose", expanded_ct)
+            },
+        orient='index', columns=["value"])
 
+    return expanded_ct, file_suffix, quality_text, debug_df
+
+def map_trip_data(df, study_type, dynamic_labels, dic_re, dic_pur):
```

Comment:
Is there a more specific variable name that could be used in places of `df`?

Reply:
I have updated the variable name from `df` to `expanded_trip_df`, since the parameter is placeholder for both `expanded_` labeled trips, and inferred trips.